queries.yaml

```diff
@@ -542,6 +552,23 @@ post_process:
       source_layer: roads
       properties: [layer]
 
+  # supersede existing osm buildings with data from landmark geojson layer
```

Comment:
landmarks mark buildings with a superseded property so that when the two overlap the downstream consumer of the tiles will be able to tell